hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieBaseFileGroupRecordBuffer.java

```diff
@@ -319,60 +311,6 @@ protected Option<T> merge(Option<T> older, Map<String, Object> olderInfoMap,
     return Option.empty();
   }
 
-  /**
```

Comment:
Moved these out of the base record buffer. extractRecordPositions is specific to position based buffer and shouldskip is only used there as well